syntrax/src/main/java/org/asciidoctor/diagram/syntrax/Syntrax.java

```diff
@@ -0,0 +1,55 @@
+package org.asciidoctor.diagram.syntrax;
+
+import org.asciidoctor.diagram.*;
+import org.atpfivt.jsyntrax.InputArguments;
+import org.atpfivt.jsyntrax.styles.StyleConfig;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static org.atpfivt.jsyntrax.Main.generateSVG;
+import static org.atpfivt.jsyntrax.Main.getStyleConfig;
+
+public class Syntrax implements DiagramGenerator {
+    public static final MimeType DEFAULT_OUTPUT_FORMAT = MimeType.SVG;
+
+    @Override
+    public String getName() {
+        return "syntrax";
+    }
+
+    @Override
+    public ResponseData generate(Request request) throws IOException {
+        MimeType format = request.headers.getValue(HTTPHeader.ACCEPT);
+        if (format == null) {
+            format = DEFAULT_OUTPUT_FORMAT;
+        }
+
+        if (!format.equals(MimeType.SVG)) {
```

Comment:
How about supporting PNG as well? Look how it's made in Main method if Jsyntrax, just one extra call is needed to convert SVG to PNG.

Reply:
new method and test added. 
SVGTranscoder.svg2Png(result) throws TranscoderException, which is caught and a new IOException exception is thrown